modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIMANAGER4081PaginationCountTestCase.java

```diff
@@ -189,6 +189,6 @@ public void destroy() throws Exception {
                 restAPIPublisher.deleteAPI(createdAPIs.get(j).getId());
             }
         }
-        super.cleanUp();
+        super.cleanUpUsingRest();
```

Comment:
Let's use the old cleanUp method as I have modified it to support latest rest api calls. Sorry for the confusion.

Reply:
Fixed